### DISCUSSION TITLE
shall we add permission control for contact point and notification policy

### DISCUSSION THREAD

**Discussion by bossjie** [grafana/grafana#65879] [Alerting]
as we have permission setting for alert rules to defend wild changing by unallowed users.
how about contact point and nofitication policy?  normal editors now are allowed to change/delete them
aka  protect partial cannot cover the protection.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82769. Feel free to continue the discussion around this there. Thank you!
